WebRoot/js/zimbraAdmin/accounts/view/ZaAccountXFormView.js

```diff
@@ -2904,12 +2913,10 @@ textFieldCssClass:"admin_xform_number_input"}
                                         trueValue:"TRUE", falseValue:"FALSE"
                                     },
                                     {ref:ZaAccount.A_zimbraPrefMailForwardingAddress, type:_TEXTFIELD_,width:"350px",
-
-labelCssClass:"xform_label", cssClass:"admin_xform_name_input",
+                                        labelCssClass:"xform_label", cssClass:"admin_xform_name_input",
```

Comment:
As long as we're inserting data "safely" we don't need to encode.  In the wrong case, it's possible (ref: https://stackoverflow.com/questions/19636775/are-html-textbox-value-attributes-safe-from-xss-attacks ). I'll not dig any further for now, but just wanted to be sure we're on the same page.